en/orm/table-objects.rst

```diff
@@ -166,17 +166,17 @@ buildRules
 
 .. php:method:: buildRules(Event $event, RulesChecker $rules)
 
-The ``Model.buildRules`` event is fired before after a rules instance has been
-created and the table's ``beforeRules()`` method has been called.
+The ``Model.buildRules`` event is fired after a rules instance has been
+created and before the table's ``beforeRules()`` method has been called.
```

Comment:
Ok sorry to waffle on opening or closing this PR but the documenation is confusing and I think it's best to clarify the language.